src/main/scala/io/smartcat/twicas/util/Conf.scala

```diff
@@ -0,0 +1,92 @@
+package io.smartcat.twicas.util
+
+import com.typesafe.config.{Config, ConfigFactory}
+
+import scala.collection.JavaConverters._
+
+object Conf {
+  lazy val textColumn: String = root.getString("text_column")
```

Comment:
More of a style comment of sorting this by config tree precedence

`
private val config = ConfigFactory.load()
private lazy val root = config.getConfig("twicas")

lazy val textColumn: String = root.getString("text_column")
`